EAGLE Libraries/HyTechDevices.lbr

```diff
@@ -25662,6 +25903,101 @@ Dimensions: 5 mm x 20 mm
 </device>
 </devices>
 </deviceset>
+<deviceset name="YC248-JR-0710KL" prefix="R">
```

Comment:
The device has multiple variants that all don't need to exist, look at Leo's CAT16 part I think for a decent single size for the array.